API.Tests/Parser/MangaParserTests.cs

```diff
@@ -77,6 +78,9 @@ public MangaParserTests(ITestOutputHelper testOutputHelper)
     [InlineData("Манга Глава 2", "0")]
     [InlineData("Манга Тома 1-4", "1-4")]
     [InlineData("Манга Том 1-4", "1-4")]
+    [InlineData("조선왕조실톡 106화", "106")]
```

Comment:
Try to keep all the Korean test cases together

Reply:
Resolved.